plugins/PluginLoader.py

```diff
@@ -0,0 +1,30 @@
+import importlib
+import os, inspect
+import importlib.util
+
+def load_plugins():
+    '''Loads all the plugins inside '/plugins' folder and then instantiate them,
+    returning it's objects.
+    '''
+    plugin_folder = "./plugins"
+    possible_plugins = os.listdir(plugin_folder)
+    plugins = []
+    plugins_instances = []
+
+    for plugin in possible_plugins:
+        if plugin.endswith(".py") and plugin != 'PluginLoader.py':
+            plugins.append(plugin)
+            
+    for p in plugins: 
+        location = plugin_folder + '/' + p
+        if p[-3:] != '.py' or os.path.isfile(location) != True:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)

Reply:
done in 'os.path.isfile(location)' check, don't know if it's necessary in "p[-3:] != '.py'" because it's a direct comparison...

https://github.com/caioluders/DPWO/pull/19/commits/bcbe2fd37defbc4b8a328b4ded3702b350269d02